testing/run_tests.py

```diff
@@ -1013,7 +1018,7 @@ def run_engine_tasks_in_parallel(tasks):
   # processes launched for the queue reader and thread wakeup reader).
   #
   # See: https://bugs.python.org/issue26903
-  max_processes = multiprocessing.cpu_count()
+  max_processes = 1  # multiprocessing.cpu_count()
```

Comment:
Is this intentional?

Reply:
No, I'll revert this. This was to try to isolate an issue on ci.